CMakeLists.txt

```diff
@@ -44,6 +44,26 @@ set(TPL_LIBRARIES "")
 # This is a list of linker flags to be used with TPLs for all targets
 set(TPL_LINKER_FLAGS "")
 
+# SAAMGE_PARALLEL
```

Comment:
@barker29 @gelever Is the order of adding TPL important? I had no problem of compiling the smoothG on a Macbook (before this commit), but on my tux box I cannot compile some of the the examples (undefined reference to ~socketstream). If I add saamge prior to mfem, the linking errors are gone (note saamge depends on mfem as well). But I am not sure if this is the actual problem (or I just build saamge in a wrong way).

Reply:
As far as I can tell, the order does matter.  It should  be in descending order, where thing up top depend on things below them.  Since saamge depends on mfem, it should be placed before mfem.

Reply:
I see, then I will keep this commit. Thanks!